modules/quantcastBidAdapter.js

```diff
@@ -92,7 +92,8 @@ export const spec = {
         },
         bidId: bid.bidId,
         gdprSignal: gdprConsent.gdprApplies ? 1 : 0,
-        gdprConsent: gdprConsent.consentString
+        gdprConsent: gdprConsent.consentString,
+        prebidJsVersion: pbjs.version
```

Comment:
Please don't assume pbjs is the object variable... use `$prebid.version$` instead of `pbjs.version`.